app/helpers/welcome_guide_helper.rb

```diff
@@ -0,0 +1,97 @@
+module WelcomeGuideHelper
+  # Contains logic that we'll used to decide what 'mode' the Welcome Guide will
+  # be shown in. The intention is for it to contain custom content for
+  # candidates who meet certain critera - e.g., a Maths final year student
+  class WelcomeGuideModeSelector
+    attr_reader :consideration_journey_stage_id, :preferred_teaching_subject, :committment_level, :degree_status_id
+
+    def initialize(mailing_list_session = {})
+      @committment_level              = mailing_list_session["committment_level"]
```

Comment:
Typo - commitment not committment?

Is this predefined so not changeable, or something new?

Reply:
Ah, well spotted. Fixed and force pushed - thanks